ext/README.md

```diff
@@ -3,6 +3,35 @@
 CEL extensions are a related set of constants, functions, macros, or other
 features which may not be covered by the core CEL spec.
 
+## Encoders
+
+Encoding utilies for marshalling data into standardized representations.
+
+### Base64.Decode
+
+Decodes a base64-encoded string to an unencoded string value.
+
+This function will return an error if the input string is not base64-encoded.
+
+    base64.decode(<string>) -> <string>
+
+Examples:
+
+    base64.decode('aGVsbG8=')  // return 'hello'
+    base64.decode('aGVsbG8')   // error
+
+### Base64.Encode
+
+Encodes a bytes or string input to a base64-encoded string.
```

Comment:
Specify that the string-accepting version converts to utf-8 bytes first. Consider accepting only bytes input and rely composition with explicit `bytes()` conversion, as this might be less error-prone.

Reply:
I think I'll still provide functions for both `string` and `bytes` inputs for simplicity on the encoding side since the composition would suffer from the same issues as the `string` based overload.